packages/superset-ui-legacy-plugin-chart-table/src/ReactDataTable.tsx

```diff
@@ -162,11 +162,10 @@ export default function ReactDataTable(props: DataTableProps) {
     if (key === '__timestamp') {
       return formatTimestamp(val);
     } else if (typeof val === 'string') {
-      // It's fine to return raw value react will handles HTML escaping
-      return val;
+      return filterXSS(val, { stripIgnoreTag: true });
```

Comment:
Could we use `dompurify` like before? 
`dompurify` seems to be more active (last updated < 1 month + 1 issue) compared to `xss` (last updated 1 year + 30 issue) with larger npm download
https://www.npmtrends.com/dompurify-vs-xss

Reply:
Both have decent number of followers but `js-xss` is much faster: 
https://www.measurethat.net/Benchmarks/Show/7485/0/js-xss-vs-dompurify

There is actually a noticeable speed difference when I tested in Superset with Chrome profiling tool.

Reply:
I am supportive if it shows significant speed improvement. If you could note down the numbers maybe we can save that for announcement email.

BTW, is the test for `js-xss` from the link above wrong? It seems to call `sanitize`.
![image](https://user-images.githubusercontent.com/1659771/75822388-bc89b980-5d54-11ea-919d-550a72d9192f.png)

Reply:
My bad. Use this instead: https://www.measurethat.net/Benchmarks/Show/7485/4/dompurify-vs-js-xss

js-xss is indeed slower than [sanitize.js](https://github.com/gbirke/Sanitize.js), but still faster than dompurify, though.

Reply:
sgtm. Thanks for being thorough.